flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/table.scala

```diff
@@ -891,7 +891,13 @@ class Table(
     *
     * @param sink The [[TableSink]] to which the [[Table]] is written.
     * @tparam T The data type that the [[TableSink]] expects.
+    *
+    * @deprecated Will be removed in a future release. Please register the TableSink and use
+    *             [[insertInto()]].
```

Comment:
For the complete javadoc, we need to remove "[[", 
[[insertInto()]] -> insertInto()
![image](https://user-images.githubusercontent.com/22488084/46772885-efc94400-cd2c-11e8-81e2-9c488fa20785.png)
![image](https://user-images.githubusercontent.com/22488084/46772899-01aae700-cd2d-11e8-96f3-842f65058d31.png)